src/appmixer/utils/csv/CSVProcessor.js

```diff
@@ -369,7 +369,6 @@ module.exports = class CSVProcessor {
      */
     async addRow(newRow, closure) {
 
-        const lock = await this.context.lock(this.fileId);
```

Comment:
why was the lock removed from the CSVProcessor? I am not sure this is correct. The lock is now only in the AddRow. What if other component calls CSVProcessor.addRow? Looks like it is called only in AddRow. But it breaks the pattern, that all the CSVProcess methods do the locking inside. 

Another thing, isn't this issue in the other components, like AddColumn which uses EditCSV, where the lock isn't being extended, either ...